### PR TITLE
Add ofd() to OSX plugin to open finder on current directory

### PR DESCRIPTION
**Pull request by mssalvatore**

### CODE REVIEW

File: plugins/osx/osx.plugin.zsh [hunk 1]
```diff
@@ -151,6 +151,14 @@ function pfd() {
 EOF
 }
 
+function ofd() {
+  osascript 2>/dev/null <<EOF
+    tell application "Finder" 
+      open POSIX file "`pwd`"
```
Comment (mcornella) [inline plugins/osx/osx.plugin.zsh]: Use `$PWD` instead of running `pwd` here.

**Comment by mcornella** [comment]
Oh wait, I have a feeling that `open_command .` should be enough to open Finder without having to write AppleScript. Can you check?

**Comment by mssalvatore** [comment]
Yeah, looks like it covers it. I'll make a new commit.

**Comment by mcornella** [comment]
Thanks! A simple alias should do the trick. I'll merge it right away.

**Comment by mssalvatore** [comment]
Do you want me to add the alias down at the bottom? What's the convention for oh-my-zsh?

Also, what's the procedure for updating the osx section of https://github.com/robbyrussell/oh-my-zsh/wiki/Plugins with the new alias?

**Comment by mcornella** [comment]
Wherever you feel it looks better, there's not a solid styleguide for that. As to the wiki, you can add it right away. But that should really be a plugin README, so don't worry about it.

**Comment by mssalvatore** [comment]
New pull request: https://github.com/robbyrussell/oh-my-zsh/pull/5165

**Pull request merged**